.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,7 +14,7 @@ Remember, you can preview this before saving it.
 ### Contributor checklist:
 
 - [ ] My commits are in nice logical chunks with [good commit messages](http://chris.beams.io/posts/git-commit/)
-- [ ] My changes are [rebased](https://blog.axosoft.com/golden-rule-of-rebasing-in-git/) on the latest [`clearnet`](https://github.com/oxen-io/session-desktop/tree/clearnet) branch
+- [ ] My changes are [rebased](https://blog.axosoft.com/golden-rule-of-rebasing-in-git/) on the latest [`unstable`](https://github.com/oxen-io/session-desktop/tree/unstable) branch
```

Comment:
The link needs to be updated to `https://github.com/session-foundation/session-desktop/tree/unstable`

Reply:
Actually looks like a few of the docs links need to be updated. Doesn't have to be part of this PR though.

Reply:
<img width="269" alt="Screenshot 2024-12-19 at 07 03 24" src="https://github.com/user-attachments/assets/e497bbaa-3d26-4ee1-ae1b-8a0e50d16755" />